config/routes.rb

```diff
@@ -86,6 +86,7 @@
       get :completed_simulations_count
       get :experiment_stats
       get :experiment_moes
+      get :experiment_results
```

Comment:
Słuszna uwaga, jeśli nie zajęło by Ci to długo to możesz zrobić aliasy stats -> experiment_stats itd. i zmienić odpowiednie wywołania w GUI na nowe. Chcę tak dlatego, że nie chcemy utracić na razie kompatybilności API restowego.